backend/Dockerfile

```diff
@@ -9,7 +9,9 @@ WORKDIR /src
 COPY *.sln .
 COPY api/*.csproj api/
 COPY entities/*.csproj entities/
-COPY test/*.csproj test/
+COPY tests/core/*.csproj tests/core/
```

Comment:
Wish there was a simple way to copy all projects in all sub-directories.

Reply:
One suggestion - perhaps we should create an additional solution `*.sln` for the testing projects? That way we would be able to clean up the build solution and wouldn't need to be breaking docker builds everytime we refactor the tests. Thoughts?

Reply:
Without them being part of the solution they wouldn't be visible to the current solution.

Reply:
That was the idea. Having one solution for code and one solution for tests. Up to you what you prefer. Just remember to run it I. Docker after every refactoring of tests